published/087-2020-04-28.md

```diff
@@ -0,0 +1,41 @@
+
+## go-ipfs 0.5.0 is officially here 🚀
+We are very excited to announce the latest release of go-ipfs 0.5.0. This release is a major step forward on our journey toward a more efficient, secure, and resilient web to preserve and grow humanity’s knowledge. [Read the full announcement on the blog for more details]( ).
+
+## Join us Friday for the 0.5 Launch Meetup!
+Join the IPFS community for a virtual meetup to dive deeper into what’s in the latest 0.5 release, how to take advantage of these improvements, and how IPFS is being used in the wild today. This event will take place **Friday, May 1, starting at 5pm UTC / 1pm ET / 10am PT**. Meet the IPFS core developers and many of the builders that are deploying IPFS into production, as well as a great line-up of speakers in-store, and also lots of time for Q&A and demos! [Please RSVP to attend](https://www.meetup.com/San-Francisco-IPFS/events/270212268/).
+
+## The latest on IPFS Mobile Design Research
+For IPFS to succeed, it needs to be where people will use it, and for the vast majority around the world that is on their smartphones. That’s why we’ve teamed up with Jim Kosem to understand how people are building, using and thinking about P2P and IPFS on mobile today in order to create tools and guidelines. [In his latest post for the IPFS blog](https://blog.ipfs.io/2020-04-24-ipfs-mobile-design-research-findings/), Jim shares a summary of his findings. 
+
+
+## Highlights from this week
+* [js-ipfs 0.42.0](https://blog.ipfs.io/2020-04-14-js-ipfs-0-42/) was also recently released 
+* Introducing [3ID Connect](https://medium.com/3box/introducing-3id-connect-531af4f84d3f), from 3Box
+* ENS unveils easier way to [build decentralized websites on Ethereum](https://decrypt.co/26246/ens-unveils-easier-way-to-build-decentralized-websites-on-ethereum)
+* Fission tried [experimenting with HLS Video Streaming and IPFS](https://blog.fission.codes/experimenting-with-hls-video-streaming-and-ipfs/)
+* TUTORIAL: [Deploy your website on IPFS: Why and How](https://ipfs.tarunbatra.com/blog/decentralization/Deploy-your-website-on-IPFS-Why-and-How/)
+
+
+## People are building the coolest stuff with IPFS
+* The Ethereum Name Service (ENS) is now available [for the Energy Web](https://medium.com/energy-web-insights/ethereum-name-service-ens-is-now-available-for-the-energy-web-a8d884138790)
+* Introducing [Pravah.io](https://pravah.io/): Data exchange for smart cities that uses libp2p
+* Putting the [Query back into Qri](https://medium.com/qri-io/putting-the-query-back-into-qri-24aa46bf3cbe)
+* Unstoppable Domains in MEW: [Get a .Crypto Home for Your Assets and Content](https://medium.com/@myetherwallet/unstoppable-domains-in-mew-get-a-crypto-home-for-your-assets-and-content-5d4847b39f04) 
+* Design a Web3 protocol and provide a courtesy node for Factom, IPFS, and ETH with [api.bitwrap.io](https://api.bitwrap.io/ipfs/QmUraA27jv1KVWJEPHo8iJj1cd2744M3WBAyWTeNXJPj1d)
+
```

Comment:
```suggestion
* Create a [Point-of-sale application with Algorand](https://developer.algorand.org/solutions/creating-point-sale-application-algorand-blockchain/) and IPFS!
```

Reply:
I'm going to have to add that to next week's since the email version went out a couple of hours ago :/